contracts/Options/SimpleOptionOnShare.sol

```diff
@@ -10,7 +10,7 @@ import "@openzeppelin/contracts/utils/math/Math.sol";
 import "@openzeppelin/contracts/utils/math/SafeMath.sol";
 import "./OptionContract.sol";
 import "../DataFeeder/ReferenceLevel.sol";
-import "../StableShares/ERC20StableShare.sol";
+import "../StableAsset/ERC20StableAsset.sol";
```

Comment:
Now we can settle in terms of StableAsset I think we can get ris of the this whole contract type as it is now the same as SimpleOption